components/feature/push/src/main/java/mozilla/components/feature/push/AutoPushFeature.kt

```diff
@@ -128,8 +128,6 @@ class AutoPushFeature(
      * This should only be done on an account logout or app data deletion.
      */
     override fun shutdown() {
-        service.stop()
```

Comment:
This is for https://github.com/mozilla-mobile/android-components/issues/5263.

With multiple features that rely on push, we shouldn't shut down the service when only the feature needs it. In Fenix, for example, Leanplum push service should still be allowed to function.

Reply:
Okay, makes sense. I would document this right where you `start` the service, to explain why there isn't a matching `stop`. The contract here is that we `start` it if it's needed, but don't `stop` it when we don't need it, since other components may be around that rely on the service.

This also means that we never `stop` this service at all, right?